connectors/golden-tables/src/test/scala/io/delta/golden/GoldenTables.scala

```diff
@@ -345,12 +358,15 @@ class GoldenTables extends QueryTest with SharedSparkSession {
     assert(new File(log.logPath.toUri).mkdirs())
 
     val file = AddFile("abc", Map.empty, 1, 1, true)
+    val metadata = Metadata(
```

Comment:
A `null` schemaString throws an error in Kernel since the field is non-nullable. Had to regenerate tables with a non-null value.